src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let RESULT = 0;
```

Comment:
```suggestion
  let result = 0;
```
it's not a constant, so better to name it in camelCase